tests/unit_test/data_processor/data_processor_test.py

```diff
@@ -117,6 +117,113 @@ async def _read_and_get_data(path: str):
 
         return _read_and_get_data
 
+    def test_add_kairon_with_invalid_similarity_threshold(self):
```

Comment:
add test case validating for system_prompt, context prompt with empty string. ValidationError should be thrown in both cases.
Also, add test case that you are able to add this action only when llm faq is enabled in bot settings